lib/builders/document/blockGridValueBuilder/blockGridContentDataBuilder.ts

```diff
@@ -0,0 +1,51 @@
+﻿import {BlockGridValueBuilder} from "./blockGridValueBuilder";
```

Comment:
There are duplicate classes for ExposeBuilder and contentDataValueBuilder, would it be possible to set them up as a base class instead?